### ISSUE TITLE
Add instructions to build for secondary arch on Haiku 32bit

### ISSUE THREAD

**Issue by nexus6-haiku** [Genio-The-Haiku-IDE/Genio#71]
Use setarch x86 make as a build command in project settings.

_https://github.com/Genio-The-Haiku-IDE/Genio/issues/69_

**Comment by Begasus** [comment]
Worked fine!

![Genio-build-x86](https://github.com/Genio-The-Haiku-IDE/Genio/assets/16057090/29bfc400-7842-4f9e-a5d1-adb0aaf1148d)

**Comment by jackburton79** [comment]
@humdingerb could we add this to the html documentation ?

**Comment by humdingerb** [comment]
Will do.
So it's really `setarch x86 make -j8`, for example, not separated by `&&` or `;` etc?

Is that `setarch x86` a special case, or can anything can be added there? How would those commands be parsed and separated without those `&&` or `;` etc?

**Comment by jackburton79** [comment]
Yes, not separated. setarch x86 has been tested and it works fine. 
IIRC that has something to do with how we launch the command in there. Separating with "&& or ;" would launch it differently with bash, so it would fail.
Maybe @Freaxed knows better.

**Comment by humdingerb** [comment]
Alright, then we keep it specific to "setarch x86" and not generalize it to "prepend any command" or something.

**Comment by jackburton79** [comment]
It's done in the documentation
